scripts/exglobal_atmos_analysis.sh

```diff
@@ -299,6 +299,7 @@ AEROINFO=${AEROINFO:-${FIXgfs}/gsi/global_aeroinfo.txt}
 SCANINFO=${SCANINFO:-${FIXgfs}/gsi/global_scaninfo.txt}
 HYBENSINFO=${HYBENSINFO:-${FIXgfs}/gsi/global_hybens_info.l${LEVS}.txt}
 OBERROR=${OBERROR:-${FIXgfs}/gsi/prepobs_errtable.global}
+BLACKLST=${BLACKLST:-${FIXgfs}/gsi/blacklist_tq_soilda.txt}
```

Comment:
see https://github.com/NOAA-EMC/GSI-fix/pull/35/files#r2052345831

Reply:
Done the change. Thanks.